velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -751,6 +753,89 @@ TpchPlan TpchQueryBuilder::getQ14Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ16Plan() const {
+  std::vector<std::string> partColumns = {
+      "p_brand", "p_type", "p_size", "p_partkey"};
+  std::vector<std::string> supplierColumns = {"s_suppkey", "s_comment"};
+  std::vector<std::string> partsuppColumns = {"ps_partkey", "ps_suppkey"};
+
+  const auto partSelectedRowType = getRowType(kPart, partColumns);
+  const auto& partFileColumns = getFileColumnNames(kPart);
+  const auto supplierSelectedRowType = getRowType(kSupplier, supplierColumns);
+  const auto& supplierFileColumns = getFileColumnNames(kSupplier);
+  const auto partsuppSelectedRowType = getRowType(kPartsupp, partsuppColumns);
+  const auto& partsuppFileColumns = getFileColumnNames(kPartsupp);
+
+  auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
+  core::PlanNodeId partScanNodeId;
+  core::PlanNodeId supplierScanNodeId;
+  core::PlanNodeId partsuppScanNodeId;
+
+  auto part = PlanBuilder(planNodeIdGenerator, pool_.get())
+                  .tableScan(
+                      kPart,
+                      partSelectedRowType,
+                      partFileColumns,
+                      {"p_size in (49, 14, 23, 45, 19, 3, 36, 9)"},
+                      "p_type NOT LIKE 'MEDIUM POLISHED%'")
+                  .capturePlanNodeId(partScanNodeId)
+                  // neq filter is currently not supported as a subfieldFilter
```

Comment:
This is specific to the Parquet Reader, so that should be specified first. The following line should be sufficient.
`Neq is unsupported as a tableScan subfield filter for Parquet source.`